lib/hanami/cli/generators/gem/app/types.erb

```diff
@@ -1,10 +1,11 @@
-# auto_register: false
```

Comment:
@timriley Do we expect this to be auto-registered in app container?

Reply:
Nope, that won't be a concern, since our generator is writing it into `lib/`, not `app/`:

```ruby
fs.write("lib/#{app}/types.rb", t("types.erb", context))
```

`lib/` isn't touched by our container auto-registration at all, so we don't need this comment 😄